sip/execution_framework_interface/docker_api/README.md

```diff
@@ -0,0 +1,41 @@
+# Docker Client API
+
+## Introduction
+
+The SIP Execution Framework provides a Docker Client Python API to run
+docker services by various Execution Control Service components.
+
+The client library is written as a of Python modules which are structured as a 
+set of higher level modules use by various Execution Control Services. 
+The client lets you run docker commands from within Python - create services, 
+delete services, manage nodes etc
+
+## Quick-start
+
+To use the client, simply copy the client script to the service directory.
+
+import the client and initialise the client.
+
+```bash
+from ..docker_client import DockerClient
+
+DC = DockerClient()
+```
+
+## Testing
+
+Linting and unit test are run automatically by SIP CI/CD system but can also be 
```

Comment:
In order for the unit tests to be run by the CI system, you'll need to add your test line to `.travis.ci` in the top level sip folder. Adding something like

```yaml
- pytest --pylint --codestyle -s -v --pylint-rcfile=./.pylintrc sip/execution_framework_interface/docker_api
```

to the end of the `before_script` section should work for now.